src/utils/oauth/config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "providers": {
+    "github": {
+      "details": {
+        "dev": {
+          "clientId": "1440dd4c1d1c4c0fa124",
+          "clientSecret": "0723a2b5bfef27efc8b2d26d837ead239fa0b0e6",
+          "redirectUrl": "http://localhost:3000/redirect",
+          "scope": "repo, user:email, read:user, read:org"
+        },
+        "deploy": {
```

Comment:
Add documentation.
Example: /** @see {@link LINK_TO_ORGANIZATION |Organization} for further information. */

Reply:
Comments are not permitted in config.json!